downloader/src/test/kotlin/VersionControlSystemTest.kt

```diff
@@ -116,6 +116,19 @@ class VersionControlSystemTest : WordSpec({
             actual shouldBe expected
         }
 
+        "properly split extra path components" {
+            val actual = VersionControlSystem.splitUrl(
+                    "ssh://git@github.com/EsotericSoftware/kryo.git/kryo-shaded"
+            )
+            val expected = VcsInfo(
+                    "Git",
+                    "ssh://git@github.com/EsotericSoftware/kryo.git",
+                    "",
+                    "kryo-shaded"
```

Comment:
Consider using named arguments for readability.

Reply:
Will do that in a new commit on this branch.